File: program/stack.c
#include <stdio.h>
#include <cs50.h>

float sum(float x,float y){
    float result=x+y;
    printf("%f\n",result);
}
void int main(){
float num1=get_float("enter your first number: ");
float num2=get_float("enter your secound number: ");
sum(num1,num2);

}

File: program/stack.py
while True:
    num1=(input("enter first number"))
    num2=(input("enter your second number"))
    if num1 >=0 and num2 >=0:
        break

def sum_number(num1,num2):
    result=int(num1)+int(num2)
    return result

x=sum_number()
print(x*10)


